vitest.shared.config.ts

```diff
@@ -5,6 +5,7 @@ import { defineConfig } from "vitest/config";
 
 export default defineConfig({
   test: {
+    testTimeout: 180000,
```

Comment:
I know this is copying the existing test timeout right? But have we considered starting smaller and bumping the timeout up as needed?

Not blocking, but I would prefer the smallest value that works to start with (if we can get away with it)

Reply:
Is this 180000 in seconds?

Reply:
Yes, this is definitely a point of concern. The live pipeline succeeded with `18000` because I missed a 0, so I think we *can* get away with starting at 1/10th of it. We could definitely start there 😉

Reply:
In the past I started with the lowest timeout that I could get away with and then doubled it whenever I had to bump the timeout. This way you aren't playing whack-a-mole with timeouts 😄 

Just one option, I think my only ask is to start with a smaller timeout if possible 👍

Reply:
@HarshaNalluru It's in milliseconds

Reply:
Updated to `18000`